pkg/blobby/testutil/test_harness.go

```diff
@@ -138,33 +155,58 @@ func (o GetOp) String() string {
 }
 
 func (o GetOp) Run(t *testing.T, ctx context.Context) error {
-	expected, _, err := o.h.model.Get(ctx, o.k)
+	val, stats, err := o.h.sut.Get(ctx, o.k)
 	if err != nil {
-		return fmt.Errorf("fakeBlobby get: %v", err)
+		// check if it's a NotFound error - that's expected for deleted/missing keys
+		if errors.Is(err, &api.NotFound{}) {
```

Comment:
discuss: this is a pretty nasty interface for callers to check whether the Get returned a NotFound or not. that seems like a common enough error that we should maybe special-case it. give me some suggestions for a better interface and we will pick one (or not).